uptane-standard.md

```diff
@@ -468,6 +476,8 @@ To be available to install on clients, all images on the repository MUST have th
 * The size of the image in bytes
 * One or more hashes of the image file, along with the hashing function used
 
+This list MAY also include an RXSWIN identifier.
```

Comment:
Should be removed, I think, like @patrickvacek noted

Reply:
That is fair, I will remove throughout the document.
I would like for RXSWIN to be included in the implementation guide/considerations document for clear guidance on how international standards (UN WP 29) may fit within Uptane.